src/model/window-guardian.ts

```diff
@@ -99,6 +99,22 @@ export const makeGuardianBrowserCAPI = (CAPI: CAPIType): CAPIBrowserType => {
         [] as CalloutBlockElement[],
     );
 
+    const qandaAtomsWithIndex: QABlockElement[] = CAPI.blocks[0].elements.reduce(
```

Comment:
As per the [Rule of Three](https://blog.codinghorror.com/rule-of-three/) it may be worth abstracting this into a `blockElementWithIndex` function now, probably in lib so it's testable.

```
const blockElementWithIndex = (blocks, blockType, indexName) => {
	// same body
}
```

?